app/controllers/APIController.scala

```diff
@@ -0,0 +1,35 @@
+package controllers
+
+import models.StatusInfo
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import javax.inject._
+import play.api._
+import play.api.mvc._
+import play.api.libs.json._
+
+/**
+ * This controller creates an `Action` to handle HTTP requests to the
+ * application's home page.
+ */
+@Singleton
+class APIController @Inject()(val controllerComponents: ControllerComponents, env: Environment) extends BaseController {
+
+  /**
+   * method that returns a json object with the following information:
+   * { "service" : "pass-bakery"
+   *   "environment" : string (production or development run, based on play fw env config)
+   *   "serverTime" : string (local time from dateTime library in ISO 8601 format)
+   * }
+   */
+
+  def getStatus() = Action { implicit request: Request[AnyContent] =>
+    implicit val statusWrites: OWrites[StatusInfo] = Json.writes[StatusInfo]
```

Comment:
Read about [companion objects](https://docs.scala-lang.org/overviews/scala-book/companion-objects.html) in scala. You should be able to move this line inside an object of the name `StatusInfo` that lives inside the `StatusInfo` file.

Reply:
The implicit line has been moved to a companion object StatusInfo inside the case class StatusInfo